src/applications/vaos/appointment-list/components/CommunityCareAppointmentDetailsPage.jsx

```diff
@@ -145,10 +148,17 @@ function CommunityCareAppointmentDetailsPage({
         </button>
       </div>
 
-      <div className="vads-u-margin-top--2 vaos-appts__block-label vads-u-background-color--primary-alt-lightest vads-u-padding--2p5">
-        Contact this provider if you need to reschedule or cancel your
+      <AlertBox
+        status={ALERT_TYPE.INFO}
+        className="vads-u-display--block"
+        backgroundOnly
+      >
+        <h2 className="vads-u-font-size--h3 vads-u-font-size--base vads-u-margin-top--0">
```

Comment:
Was there a reason for putting the header here vs using the `headline` and `level` props?

Reply:
Nope. Just cut and paste from some other code. I will change to use props. The default level is 3 with is fine.